addons/binding/org.openhab.binding.denonmarantz/META-INF/MANIFEST.MF

```diff
@@ -1,32 +1,36 @@
 Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
 Bundle-ManifestVersion: 2
 Bundle-Name: DenonMarantz Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-SymbolicName: org.openhab.binding.denonmarantz;singleton:=true
 Bundle-Vendor: openHAB
 Bundle-Version: 2.3.0.qualifier
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-ClassPath: .
+Export-Package: 
+ org.openhab.binding.denonmarantz,
+ org.openhab.binding.denonmarantz.handler
 Import-Package: 
  javax.jmdns,
  org.apache.commons.io,
  org.apache.commons.lang,
  org.apache.commons.net,
+ org.eclipse.jdt.annotation;resolution:=optional,
  org.eclipse.jetty.client,
  org.eclipse.jetty.client.api,
  org.eclipse.jetty.http,
  org.eclipse.jetty.util.component,
  org.eclipse.smarthome.config.core,
  org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.config.discovery.mdns,
  org.eclipse.smarthome.core.library.types,
  org.eclipse.smarthome.core.thing,
  org.eclipse.smarthome.core.thing.binding,
  org.eclipse.smarthome.core.thing.binding.builder,
  org.eclipse.smarthome.core.thing.type,
  org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.io.transport.mdns.discovery,
  org.openhab.binding.denonmarantz,
  org.openhab.binding.denonmarantz.handler,
+ org.osgi.service.component.annotations;resolution:=optional,
```

Comment:
Are you sure that you removed `org.osgi.service.component.annotations;resolution:=optional,` and **NOT** accidentally `org.eclipse.jdt.annotation;resolution:=optional,`? Because the later one would explain your error message...

Reply:
Mea culpa, yes I did.
However, when I remove the `org.osgi.service.component.annotations;resolution:=optional,` I see the following warning in Eclipse:
`DS Annotations missing from permanent build path`

Reply:
This is because your branch is too old. I have just tested a plain checkout with the current master and everything is fine in the IDE. if I rebase your branch onto the current master and remove `org.osgi.service.component.annotations;resolution:=optional,` from the `MANIFEST.MF`, everything is alright.

So please rebase your branch onto the current master and do a push with --force. Careful though, because this will overwrite the branch on github. So you might want to do a backup of your current local repository before doing so since you said you had problems with git before.

However, I just did a `git rebase master` in your branch locally (after pulling the latest master branch of course) on my machine and everything went smooth.

Reply:
I am on the latest commit (commit dc204c6cc8fc3e1be9296c773b0c4eade3c4f7aa) of openhab2-addons, still see the warning.
Is it perhaps a setting outside of this repo? What was changed to resolve this warning?

Reply:
Nevertheless, I will apply the change and ignore the local warning (I see them for other bindings as well), perhaps related to my IDE setup.

Reply:
All of you checked out repos have to be on the newest version, i.e. "openhab1-addons  openhab2-addons  openhab-core  openhab-distro  smarthome"

Reply:
I only have `openhab2-addons` and `openhab-distro` and both are up to date (while [setting up the IDE](https://docs.openhab.org/developers/development/ide.html#installation) I only picked "openHAB Development" (required) and "openHAB2 Add-ons" from the optional set of repos).

Reply:
Confirmed warning is gone when all git repos are included in the IDE.